cmd/tray/main.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/NordSecurity/nordvpn-linux/daemon/pb"
+	"github.com/NordSecurity/nordvpn-linux/internal"
+	meshpb "github.com/NordSecurity/nordvpn-linux/meshnet/pb"
+	"github.com/NordSecurity/nordvpn-linux/tray"
+
+	"github.com/NordSecurity/systray"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+var (
+	DaemonURL = fmt.Sprintf("%s://%s", internal.Proto, internal.DaemonSocket)
+)
+
+func onExit() {
+	if tray.DebugMode {
+		now := time.Now()
+		fmt.Println("Exit at", now.String())
+	}
+}
+
+func main() {
+	conn, err := grpc.Dial(
+		DaemonURL,
+		grpc.WithTransportCredentials(insecure.NewCredentials()),
+	)
+
+	if err == nil {
+		tray.Client = pb.NewDaemonClient(conn)
```

Comment:
Similar pattern is used in the CLI. There is no problem if daemon is not running or it disconnects during operation. The client will reconnect seamlessly in the background. So I don't know what are the cases it can return an error.